packages/cli/lib/utils.js

```diff
@@ -295,3 +297,46 @@ exports.validateStringObject = function(type) {
     return true;
   };
 };
+
+/**
+ * Use readline to read text from stdin
+ * @param {*} log An optional log
+ */
+exports.readTextFromStdin = function(log) {
+  const rl = readline.createInterface({
+    input: process.stdin,
+  });
+
+  const lines = [];
+  if (process.stdin.isTTY && log) {
+    log(
+      chalk.green(
+        'Please type in a json object line by line ' +
```

Comment:
In my view, this message does not belong to `readTextFromStdin` as `readTextFromStdin` can be used to read arbitrary text, not only JSON.

I am proposing to move it back to `_readJSONFromStdin` and also remove the no-longer used `log` argument.